From 1456b5a1f7061a4b3bd522b241d2233c8856b52a Mon Sep 17 00:00:00 2001
From: Mike Baiocchi <mbaiocch@us.ibm.com>
Date: Wed, 14 Nov 2018 18:13:04 -0600
Subject: [PATCH] Update HBD/HBD_RW Partition Sizes to fix VESNIN Config
 Compile Issues

This commit rebalances the allocated size of 0x60000 for the P8
HBD/HBD_RW partitions to more properly reflect that for P8 systems
there is much more read-only attribute data (which goes into HBD)
then there is read-write data (which goes into HBD_RW).
---
 p8Layouts/defaultPnorLayoutSingleSide.xml     | 10 +++++-----
 p8Layouts/defaultPnorLayoutWithGoldenSide.xml | 20 +++++++++----------
 .../defaultPnorLayoutWithoutGoldenSide.xml    | 18 ++++++++---------
 3 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/p8Layouts/defaultPnorLayoutSingleSide.xml b/p8Layouts/defaultPnorLayoutSingleSide.xml
index 7c9c7d1..f4ca275 100755
--- a/p8Layouts/defaultPnorLayoutSingleSide.xml
+++ b/p8Layouts/defaultPnorLayoutSingleSide.xml
@@ -109,19 +109,19 @@ Layout Description
         <reprovision/>
     </section>
     <section>
-        <description>Hostboot Data (320K)</description>
+        <description>Hostboot Data (360K)</description>
         <eyeCatch>HBD</eyeCatch>
         <physicalOffset>0x6D0000</physicalOffset>
-        <physicalRegionSize>0x50000</physicalRegionSize>
+        <physicalRegionSize>0x5A000</physicalRegionSize>
         <side>A</side>
         <ecc/>
         <readOnly/>
     </section>
     <section>
-        <description>Hostboot Data : Read-Write (64K)</description>
+        <description>Hostboot Data : Read-Write (24K)</description>
         <eyeCatch>HBD_RW</eyeCatch>
-        <physicalOffset>0x720000</physicalOffset>
-        <physicalRegionSize>0x10000</physicalRegionSize>
+        <physicalOffset>0x72A000</physicalOffset>
+        <physicalRegionSize>0x6000</physicalRegionSize>
         <side>A</side>
         <ecc/>
     </section>
diff --git a/p8Layouts/defaultPnorLayoutWithGoldenSide.xml b/p8Layouts/defaultPnorLayoutWithGoldenSide.xml
index 8982677..44a0580 100755
--- a/p8Layouts/defaultPnorLayoutWithGoldenSide.xml
+++ b/p8Layouts/defaultPnorLayoutWithGoldenSide.xml
@@ -97,19 +97,19 @@ Layout Description
         <reprovision/>
     </section>
     <section>
-        <description>Hostboot Data (320K)</description>
+        <description>Hostboot Data (360K)</description>
         <eyeCatch>HBD</eyeCatch>
         <physicalOffset>0x31000</physicalOffset>
-        <physicalRegionSize>0x50000</physicalRegionSize>
+        <physicalRegionSize>0x5A000</physicalRegionSize>
         <side>A</side>
         <ecc/>
         <readOnly/>
     </section>
     <section>
-        <description>Hostboot Data : Read-Write (64K)</description>
+        <description>Hostboot Data : Read-Write (24K)</description>
         <eyeCatch>HBD_RW</eyeCatch>
-        <physicalOffset>0x81000</physicalOffset>
-        <physicalRegionSize>0x10000</physicalRegionSize>
+        <physicalOffset>0x8B000</physicalOffset>
+        <physicalRegionSize>0x6000</physicalRegionSize>
         <side>A</side>
         <ecc/>
     </section>
@@ -321,19 +321,19 @@ Layout Description
     </section>
     <!-- Golden Side (Side B) -->
     <section>
-        <description>Hostboot Data (320K)</description>
+        <description>Hostboot Data (360K)</description>
         <eyeCatch>HBD</eyeCatch>
         <physicalOffset>0x2008000</physicalOffset>
-        <physicalRegionSize>0x50000</physicalRegionSize>
+        <physicalRegionSize>0x5A000</physicalRegionSize>
         <side>B</side>
         <readOnly/>
         <ecc/>
     </section>
     <section>
-        <description>Hostboot Data : Read-Write (64K)</description>
+        <description>Hostboot Data : Read-Write (24K)</description>
         <eyeCatch>HBD_RW</eyeCatch>
-        <physicalOffset>0x2058000</physicalOffset>
-        <physicalRegionSize>0x10000</physicalRegionSize>
+        <physicalOffset>0x2062000</physicalOffset>
+        <physicalRegionSize>0x6000</physicalRegionSize>
         <side>B</side>
         <readOnly/>
         <ecc/>
diff --git a/p8Layouts/defaultPnorLayoutWithoutGoldenSide.xml b/p8Layouts/defaultPnorLayoutWithoutGoldenSide.xml
index 925f9fd..d1d5af5 100755
--- a/p8Layouts/defaultPnorLayoutWithoutGoldenSide.xml
+++ b/p8Layouts/defaultPnorLayoutWithoutGoldenSide.xml
@@ -96,19 +96,19 @@ Layout Description
         <reprovision/>
     </section>
     <section>
-        <description>Hostboot Data (320K)</description>
+        <description>Hostboot Data (360K)</description>
         <eyeCatch>HBD</eyeCatch>
         <physicalOffset>0x31000</physicalOffset>
-        <physicalRegionSize>0x50000</physicalRegionSize>
+        <physicalRegionSize>0x5A000</physicalRegionSize>
         <side>A</side>
         <ecc/>
         <readOnly/>
     </section>
     <section>
-        <description>Hostboot Data : Read-Write (64K)</description>
+        <description>Hostboot Data : Read-Write (24K)</description>
         <eyeCatch>HBD_RW</eyeCatch>
-        <physicalOffset>0x81000</physicalOffset>
-        <physicalRegionSize>0x10000</physicalRegionSize>
+        <physicalOffset>0x8B000</physicalOffset>
+        <physicalRegionSize>0x6000</physicalRegionSize>
         <side>A</side>
         <ecc/>
     </section>
@@ -320,7 +320,7 @@ Layout Description
     </section>
     <!-- Golden Side (Side B) -->
     <section>
-        <description>Hostboot Data (320K)</description>
+        <description>Hostboot Data (360K)</description>
         <eyeCatch>HBD</eyeCatch>
         <physicalOffset>0x2008000</physicalOffset>
         <physicalRegionSize>0x50000</physicalRegionSize>
@@ -329,10 +329,10 @@ Layout Description
         <readOnly>
     </section>
     <section>
-        <description>Hostboot Data : Read-Write (64K)</description>
+        <description>Hostboot Data : Read-Write (24K)</description>
         <eyeCatch>HBD_RW</eyeCatch>
-        <physicalOffset>0x2058000</physicalOffset>
-        <physicalRegionSize>0x10000</physicalRegionSize>
+        <physicalOffset>0x2062000</physicalOffset>
+        <physicalRegionSize>0x60000</physicalRegionSize>
         <side>B</side>
         <ecc/>
     </section>
-- 
2.19.1

